examples/complete/variables.tf

```diff
@@ -51,3 +51,9 @@ variable "secondary_use_vsi_security_group" {
   type        = bool
   default     = false
 }
+
+variable "enable_dedicated_host" {
+  type        = bool
+  default     = false
+  description = "Enabling this option will activate dedicated hosts for the VSIs. When enabled, the dedicated_host_id input is required. The default value is set to false. Refer [Understanding Dedicated Hosts](https://cloud.ibm.com/docs/vpc?topic=vpc-creating-dedicated-hosts-instances&interface=ui#about-dedicated-hosts) for more details"
```

Comment:
This description is not correct for the example

Reply:
Done.